spec/factories/state_file/personas/2023/az.rb

```diff
@@ -0,0 +1,275 @@
+        dob: Date.new(1940, 3, 3),
+        relationship: "GRANDPARENT",
+        months_in_home: 7,
+        needed_assistance: "no",
+        passed_away: "no"
+      )
+      intake.dependents.where(first_name: "Wendy").first.update(
+        dob: Date.new(1940, 4, 4),
+        relationship: "GRANDPARENT",
+        months_in_home: 12,
+        needed_assistance: "yes",
+        passed_away: "no"
+      )
+
+      intake.dependents.reload
+    end
+
+    has_prior_last_names { "yes" }
+    prior_last_names { "Schitt, Creek" }
+
+    tribal_member { "yes" }
+    tribal_wages { 1000 }
+
+    armed_forces_member { "no" }
+
+    charitable_contributions { "no" }
+
+    primary_state_id {
+      create :state_id,
+             id_type: 'driver_license',
+             id_number: '123456',
+             state: 'AZ',
+             issue_date: Date.new(2020, 1, 1),
+             expiration_date: Date.new(2027, 1, 1),
+             first_three_doc_num: nil
+    }
+
+    spouse_state_id {
+      create :state_id,
+             id_type: 'dmv_bmv',
+             id_number: '654321',
+             state: 'MN',
+             issue_date: Date.new(2021, 1, 1),
+             expiration_date: Date.new(2028, 1, 1),
+             first_three_doc_num: nil
+    }
+
+    payment_or_deposit_type { "direct_deposit" }
+    bank_name { "Canvas Credit union" }
+    account_type { "checking" }
+    routing_number { "302075830" }
+    account_number { "123456" }
+
+    federal_submission_id { "12345202201011234570" }
+  end
+
+  factory "leslie", class: StateFileAzIntake do
+    raw_direct_file_data { StateFile::XmlReturnSampleService.new.read("az_leslie_qss_v2") }
+    primary_first_name { "LESLIE" }
+    primary_last_name { "KNOPE" }
+    primary_birth_date { "1955-12-12" }
+
+    after(:create) do |intake|
+      intake.synchronize_df_dependents_to_database
+
+      intake.dependents.where(first_name: "April").first.update(
+        dob: Date.new(2019, 8, 8),
+        relationship: "DAUGHTER",
+        months_in_home: 12
+      )
+
+      intake.dependents.reload
+    end
+
+    after(:create) do |intake|
+      create(
+        :state_file1099_g,
+        intake: intake,
+        payer_name: "ARIZONA DEPARTMENT OF ECONOMIC SECURITY",
+        payer_tin: "270293117",
+        payer_street_address: "568 BREWER CIRCLE",
+        payer_city: "PHOENIX",
+        payer_zip: "85034",
+        recipient: "primary",
+        recipient_street_address: "321 ANDY STREET",
+        recipient_city: "PHOENIX",
+        recipient_zip: "85034",
+        unemployment_compensation: 10000,
+        federal_income_tax_withheld: 10,
+        state_identification_number: "123456",
+        state_income_tax_withheld: 10,
+        )
+    end
+
+    has_prior_last_names { "no" }
+
+    tribal_member { "no" }
+
+    armed_forces_member { "yes" }
+    armed_forces_wages { 5000 }
+
+    charitable_contributions { "no" }
+
+    primary_state_id {
```

Comment:
look into whether the factory defaults can be deleted without breaking too many tests, so that these can be slimmer